src/module.ts

```diff
@@ -47,8 +49,9 @@ export default defineNuxtModule<ModuleOptions>({
 
     // Enable dirs
     // nuxt.options.components.dirs = ["~/components/storyblok"];
-    addComponentsDir({ path: "~/storyblok", global: true, pathPrefix: false });
-
+    if(options.globalDir){
```

Comment:
If `globalDir` has a default value, is this `if` really needed?

Reply:
Yes if you set the property in the Nuxt.config to

```
globalDir = false
```

You are able to disable it.